src/Tester/StreamingTests/PullingAgentManagementTests.cs

```diff
@@ -16,30 +16,23 @@ namespace UnitTests.StreamingTests
     [DeploymentItem("ClientConfigurationForStreamTesting.xml")]
     [DeploymentItem("OrleansProviders.dll")]
     [TestClass]
-    public class PullingAgentManagementTests : UnitTestSiloHost
+    public class PullingAgentManagementTests : UnitTestSiloHostPerFixture
```

Comment:
Cool. I have played around with xUnit on another project, and after factoring out the shared context into "fixtures" then porting from MsTest to xUnit is very straight forward task :)